test/message.spec.js

```diff
@@ -0,0 +1,126 @@
+var Promise = require("bluebird");
+var expect = require("chai").expect;
+var supertest = require("supertest-promised");
+var parseXML = require("xml-parser");
+var find = require("lodash/find");
+var app = require("../app");
+var SurveyResponse = require("../models/SurveyResponse");
+var agent = supertest(app);
+
+describe("POST /message", function() {
+  beforeEach(function() {
+    return SurveyResponse.remove({})
+    .then(() => console.log("### delete all"))
+  });
+
+  it("Should return a greeting/question on first survey attempt.", function() {
```

Comment:
It's funny that you point that out because I was actually looking for a way to shorten that line and it didn't occur to me that `Should` was not necessary. Nice catch Sir!